lib/app/sequence_base.rb

```diff
@@ -54,15 +54,14 @@ class SequenceBase
       delegate :versioned_conformance_class, to: :@instance
       delegate :save_resource_ids_in_bundle, to: :@instance
 
-      def initialize(instance, client, disable_tls_tests = false, sequence_result = nil, metadata_only = false)
+      def initialize(instance, client, disable_tls_tests = false, sequence_result = nil)
         @client = client
         @instance = instance
         @client.set_bearer_token(@instance.token) unless @client.nil? || @instance.nil? || @instance.token.nil?
         @client&.monitor_requests
         @sequence_result = sequence_result
         @disable_tls_tests = disable_tls_tests
         @test_warnings = []
-        @metadata_only = metadata_only
```

Comment:
Did we get rid of the idea of "metadata only" tests?

Reply:
Yes, rather than an initial run of each test with `@metadata_only` set, each test now has an initial run in the context of an `InfernoTest` (instead of in the context of the Sequence the test belongs to. `@metadata_only` caused the `metadata` method to raise an exception after its block was evaluated so that the actual test didn't run. Now that is just how the `metadata` method in `InfernoTest` always works, while `metadata` in `SequenceBase` is a no-op.

Reply:
Yeah this is neat.